diff --git a/node_modules/nuxt-og-image/dist/runtime/nitro/routes/html.mjs b/node_modules/nuxt-og-image/dist/runtime/nitro/routes/html.mjs
index b1ad662..88d98bc 100644
--- a/node_modules/nuxt-og-image/dist/runtime/nitro/routes/html.mjs
+++ b/node_modules/nuxt-og-image/dist/runtime/nitro/routes/html.mjs
@@ -3,7 +3,6 @@ import { renderSSRHead } from "@unhead/ssr";
 import { createHeadCore } from "@unhead/vue";
 import { createError, defineEventHandler, getQuery, sendRedirect } from "h3";
 import { hash } from "ohash";
-import twemoji from "twemoji";
 import { createDefu } from "defu";
 import { fetchOptionsCached } from "../utils.mjs";
 import { useNitroOrigin, useRuntimeConfig } from "#imports";
@@ -52,13 +51,6 @@ export default defineEventHandler(async (e) => {
   if (firstFont)
     defaultFontFamily = firstFont.name;
   let html = island.html;
-  try {
-    html = twemoji.parse(html, {
-      folder: "svg",
-      ext: ".svg"
-    });
-  } catch (e2) {
-  }
   head.push({
     style: [
       {
