diff --git a/node_modules/nuxt-link-checker/dist/module.mjs b/node_modules/nuxt-link-checker/dist/module.mjs
index a81020c..d3de100 100644
--- a/node_modules/nuxt-link-checker/dist/module.mjs
+++ b/node_modules/nuxt-link-checker/dist/module.mjs
@@ -285,15 +285,16 @@ const responses = {};
 async function getLinkResponse({ link, timeout, fetchRemoteUrls, baseURL, isInStorage }) {
   if (link.includes("#") && !link.startsWith("#"))
     link = link.split("#")[0];
-  const response = responses[link];
+  const route = decodeURI(link);
+  const response = responses[route];
   if (!response) {
     if (isNonFetchableLink(link) || link.startsWith("http") && !fetchRemoteUrls || isInStorage()) {
-      responses[link] = Promise.resolve({ status: 200, statusText: "OK", headers: {} });
+      responses[route] = Promise.resolve({ status: 200, statusText: "OK", headers: {} });
     } else {
-      responses[link] = crawlFetch(link, { timeout, baseURL });
+      responses[route] = crawlFetch(link, { timeout, baseURL });
     }
   }
-  return responses[link];
+  return responses[route];
 }
 function setLinkResponse(link, response) {
   responses[link] = response;
